teemaderegister-react/src/components/SupervisorMeta.jsx

```diff
@@ -111,10 +114,7 @@ class SupervisorMeta extends PureComponent {
           >
             <h4>Topics</h4>
             <div>
-              Tallinna Ülikooli vilistlane, kes on lõpetanud nii Informaatika
-              eriala bakalaureuse tasemel kui ka läbinud magistriõppe. Õppejõuna
-              üritab ta pidevalt ennast täiendada ning kasutada uusi
-              tehnoloogiaid, rõhudes alati praktilise lõppväljundi
+              {descriptionBox}
```

Comment:
Currently shows this if description is not added


<img width="252" alt="Screenshot 2021-01-13 at 22 38 11" src="https://user-images.githubusercontent.com/4492641/104508001-400c6c00-55f0-11eb-8440-e40982b0af5b.png">

Rather show a dash "-" if no data is provided.

Reply:
just use here 
```
{description || '–'}
```